pipeline_tools/tests/data/analysis.json

```diff
@@ -1,96 +1,98 @@
+    },
+    {
+      "parameter_name": "sample_name",
+      "parameter_value": "adb6ee29-43ac-4b5e-a91c-8808d0d17321"
+    }
+  ],
+  "outputs": [
+    {
+      "describedBy": "https://schema.humancellatlas.org/type/file/5.2.1/analysis_file",
+      "file_core": {
+        "file_format": "bam",
+        "file_name": "adb6ee29-43ac-4b5e-a91c-8808d0d17321_qc.bam"
       },
-      {
-        "describedBy": "https://schema.humancellatlas.org/type/file/5.0.0/analysis_file",
-        "schema_type": "file",
-        "file_core": {
-          "file_name": "counts_matrix",
-          "checksum": "d0f7d08f7910f7980f",
-          "file_format": "counts_matrix"
-        }
+      "schema_type": "file"
+    },
+    {
+      "describedBy": "https://schema.humancellatlas.org/type/file/5.2.1/analysis_file",
+      "file_core": {
+        "file_format": "txt",
+        "file_name": "adb6ee29-43ac-4b5e-a91c-8808d0d17321_qc.alignment_summary_metrics.txt"
       },
+      "schema_type": "file"
+    },
     {
-      "describedBy": "https://schema.humancellatlas.org/type/file/5.0.0/analysis_file",
-      "schema_type": "file",
+      "describedBy": "https://schema.humancellatlas.org/type/file/5.2.1/analysis_file",
       "file_core": {
-        "file_name": "report_qc",
-        "file_format": "qc_matrix",
-        "checksum": "d0f7d18f7980f7980f"
-      }
-    }
-   ],
-  "metadata_schema" : "version_232",
-  "tasks" : [
-      {
-       "stop_time" : "2017-05-18T08:44:23-5:00",
-       "start_time" : "2017-05-18T03:44:23-5:00",
-       "task_name" : "step_one",
-       "log_err" : "bluebox://43bd7eb0-23a3-4898-b043-f7e982de281a/v1/05bd7eb0-23a3-4898-b043-f7e982de281g/v1#10xv2_step_one.err",
-       "log_out" : "bluebox://43bd7eb0-23a3-4898-b043-f7e982de281a/v1/05bd7eb0-23a3-4898-b043-f7e982de281g/v1#10xv2_step_one.out",
-       "disk_size" : "500GB",
-       "docker_image" : "https://uri-to-docker-image",
-       "cpus" : 8,
-       "memory" : "10GB",
-       "zone" : "us-east4-a"
+        "file_format": "txt",
+        "file_name": "adb6ee29-43ac-4b5e-a91c-8808d0d17321_qc.bait_bias_detail_metrics.txt"
       },
-      {
-       "start_utc" : "2017-05-18T08:44:23-5:00",
-       "stop_utc" : "2017-05-18T14:44:23-5:00",
-       "task_name" : "step_two",
-       "log_err" : "bluebox://43bd7eb0-23a3-4898-b043-f7e982de281a/v1/05bd7eb0-23a3-4898-b043-f7e982de281g/v1#10xv2_step_two.err",
-       "log_out" : "bluebox://43bd7eb0-23a3-4898-b043-f7e982de281a/v1/05bd7eb0-23a3-4898-b043-f7e982de281g/v1#10xv2_step_two.out",
-       "disk_size" : "500GB",
-       "docker_image" : "https://uri-to-docker-image",
-       "cpus" : 8,
-       "memory" : "10GB",
-       "zone" : "us-east4-a"
-      }
+      "schema_type": "file"
+    }
+  ],
+  "protocol_core": {
+    "protocol_id": "ff84b09c-2b65-4d65-9ba2-7437e9ddac8b"
```

Comment:
I'm not sure what the current state of the schema is, but I thought we were going to have the protocol id be the same for all submissions from the same version of the pipeline, whereas the process id would be unique to the workflow run.

Reply:
For the schema `https://schema.humancellatlas.org/type/protocol/analysis/7.0.0/analysis_protocol`, the analysis process was just renamed "analysis protocol" so the process_id became the protocol_id. There is a PR in the metadata repo (https://github.com/HumanCellAtlas/metadata-schema/pull/396)  that will fix the schema to include both an analysis protocol and analysis process schema.